app/_posts/2024-06-25-kuma-2-8-0.md

```diff
@@ -0,0 +1,91 @@
+* A new `MeshPassthrough` policy that allows exposing external endpoints for rich clients through the mesh and support for wildcard domains.
+* HostnameGenerator provides a way to generate custom domains for your `MeshExternalServices`, offering enhanced security.
+
+Feel free to check our [release notes](https://github.com/kumahq/kuma/releases/tag/2.8.0) for the full list of changes.
+
+## HostnameGenerator
+
+It's a brand new resource [`HostnameGenerator`](/docs/2.8.x/networking/hostnamegenerator) that enables you to generate custom domain for your `MeshExternalService`. By default Kuma is generating a domain for your `ExternalService` by adding suffix `.mesh` to the name of your `ExternalService`. With this release, you now have control over the domain.
+
+Let's take a look at this example:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: HostnameGenerator
+metadata:
+  name: example
+  namespace: kuma-system
+  labels:
+    kuma.io/mesh: default
+spec:
+  selector:
+    meshExternalService:
+      matchLabels:
+        kuma.io/origin: zone
+  template: "{{ .DisplayName }}.svc.meshext.local"
+```
+
+and now let's create a `MeshExternalService`
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshExternalService
+metadata:
+  name: mes-http
+  namespace: kuma-system
+  labels:
+    kuma.io/mesh: default
+spec:
+  match:
+    type: HostnameGenerator
+    port: 80
+    protocol: http
+  endpoints:
+  - address: httpbin.org
+    port: 80
+```
+
+In a brand new tab in the GUI, we can now see the generated domain address for our new `MeshExternalService`.
+
+![MeshExternalService view with a custom domain provided by the HostnameGenerator](/assets/images/blog/kuma_2_8_mes_hostname.png)
+
+## MeshExternalService
+The `ExternalService` resource struggled with various issues, leading us to develop the more powerful [`MeshExternalService`](/docs/2.8.x/networking/meshexternalservice/) resource. This brand-new resource enables you to add external endpoints to your mesh and overcome the limitations of the `ExternalService`. `MeshExternalService` will eventually replace `ExternalService` and is currently in the alpha phase.
+
+![MeshExternalService tab view](/assets/images/blog/kuma_2_8_mes.png)
+
+Additionally, you can see status information about the hostname and address of the `MeshExternalService`.
+
+![MeshExternalService status](/assets/images/blog/kuma_2_8_mes_status.png)
+
+## MeshPassthrough
+
+Some of the issues with `ExternalService` didn't align with the new design of `MeshExternalService`. As a result, we've introduced a new policy called [`MeshPassthrough`](https://kuma.io/docs/2.8.x/policies/meshpassthrough). This policy facilitates communication with external endpoints for rich clients and offers the flexibility to enable or disable [passthrough mode](/docs/2.8.x/networking/non-mesh-traffic/#outgoing) for individual sidecars. Learn more about passthrough mode.
```

Comment:
should passthrough mode be a link?

should we add more info about what "rich clients" are?

```suggestion
Some of the issues with `ExternalService` didn't align with the new design of `MeshExternalService`. As a result, we've introduced a new policy called [`MeshPassthrough`](https://kuma.io/docs/2.8.x/policies/meshpassthrough). This policy facilitates communication with external endpoints for rich clients and offers the flexibility to enable or disable [passthrough mode](/docs/2.8.x/networking/non-mesh-traffic/#outgoing) for individual sidecars. Learn more about [passthrough mode](?).
```